build/Helix/PrepareHelixPayload.ps1

```diff
@@ -50,6 +50,8 @@ Copy-If-Exists "$repoDirectory\Artifacts\$ArtifactName\$Configuration\$Platform\
 
 # NuGet test artifacts
```

Comment:
Please update this comment to something like
# Copy files from the 'NugetPkgTestsDrop' or 'FrameworkPkgTestsDrop' artifact dir